msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -211,6 +211,9 @@ Use the following format for a deep link that you can use in a bot, Connector, o
 
 Example: `https://teams.microsoft.com/l/meeting/new?subject=test%20subject&attendees=joe@contoso.com,bob@contoso.com&startTime=10%2F24%2F2018%2010%3A30%3A00&endTime=10%2F24%2F2018%2010%3A30%3A00&content=​​​​​​​test%3Acontent​​​​​​​​​​​​​​`
 
+> [!NOTE]
+> The search params do not support the `+` signal in place of the whitespace(` `) character. You should ensure your uri encoding code returns `%20` for spaces. Eg: `?subject=test%20subject` is good, but `?subject=test+subject` is bad.  
```

Comment:
Please rephrase:
```suggestion
> [!NOTE]
> The search parameters don't t support `+` signal in place of whitespace (` `). Ensure your uri encoding code returns `%20` for spaces for example, `?subject=test%20subject` is good, but `?subject=test+subject` is bad.
```

Reply:
Rephrased the copy as you suggested. I didn't apply your suggestion as it contained a typo on the `don't t`